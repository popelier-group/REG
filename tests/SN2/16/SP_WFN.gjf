# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.95684200   -0.00005900    0.00019700
 H                 -0.89484900   -0.90912500   -0.56635300
 H                 -0.89483300    0.94513300   -0.50377000
 H                 -0.89489300   -0.03619500    1.07075400
 Br                 1.62424400    0.00000600   -0.00001800
 Cl                -3.25989900    0.00001100   -0.00004600

frame.wfn
