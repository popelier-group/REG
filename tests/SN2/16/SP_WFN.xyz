
6

C        0.000190   -0.000056    0.842164
H       -0.566413   -0.909089    0.780171
H       -0.503722    0.945165    0.780154
H        1.070745   -0.036254    0.780216
Br       -0.000023    0.000007   -1.738922
Cl       -0.000055    0.000016    3.145221
