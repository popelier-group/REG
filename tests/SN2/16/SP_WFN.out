 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.16/Gau-74898.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.16/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=     74899.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.95684  -0.00006   0.0002 
 H                    -0.89485  -0.90913  -0.56635 
 H                    -0.89483   0.94513  -0.50377 
 H                    -0.89489  -0.0362    1.07075 
 Br                    1.62424   0.00001  -0.00002 
 Cl                   -3.2599    0.00001  -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.956842   -0.000059    0.000197
      2          1           0       -0.894849   -0.909125   -0.566353
      3          1           0       -0.894833    0.945133   -0.503770
      4          1           0       -0.894893   -0.036195    1.070754
      5         35           0        1.624244    0.000006   -0.000018
      6         17           0       -3.259899    0.000011   -0.000046
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.072951   0.000000
     3  H    1.072947   1.855314   0.000000
     4  H    1.072957   1.855297   1.855298   0.000000
     5  Br   2.581086   2.737350   2.737294   2.737501   0.000000
     6  Cl   2.303057   2.596285   2.596253   2.596378   4.884143
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000190   -0.000056    0.842164
      2          1           0       -0.566413   -0.909089    0.780171
      3          1           0       -0.503722    0.945165    0.780154
      4          1           0        1.070745   -0.036254    0.780216
      5         35           0       -0.000023    0.000007   -1.738922
      6         17           0       -0.000055    0.000016    3.145221
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         145.6808075           0.8494421           0.8494421
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.3831030512 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  5.70D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.38805537     A.U. after   12 cycles
            NFock= 12  Conv=0.41D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.60301-101.33667 -62.25611 -56.07467 -56.07343
 Alpha  occ. eigenvalues --  -56.07343 -10.09155  -9.24974  -8.46498  -7.01126
 Alpha  occ. eigenvalues --   -7.00630  -7.00630  -6.29056  -6.28640  -6.28640
 Alpha  occ. eigenvalues --   -2.40244  -2.40130  -2.40130  -2.39802  -2.39802
 Alpha  occ. eigenvalues --   -0.61265  -0.56041  -0.51965  -0.30887  -0.30887
 Alpha  occ. eigenvalues --   -0.18161  -0.12389  -0.12389  -0.09286  -0.09286
 Alpha  occ. eigenvalues --   -0.09175
 Alpha virt. eigenvalues --    0.09527   0.11520   0.15183   0.15183   0.16197
 Alpha virt. eigenvalues --    0.17519   0.18787   0.18787   0.20532   0.21396
 Alpha virt. eigenvalues --    0.21396   0.22880   0.22881   0.24695   0.29833
 Alpha virt. eigenvalues --    0.31642   0.31642   0.36949   0.39064   0.54728
 Alpha virt. eigenvalues --    0.54728   0.54865   0.58096   0.58096   0.58548
 Alpha virt. eigenvalues --    0.63327   0.63327   0.69383   0.77224   0.77225
 Alpha virt. eigenvalues --    0.78448   0.83570   0.84243   0.84243   0.84405
 Alpha virt. eigenvalues --    0.84405   0.90899   1.07135   1.07135   1.07879
 Alpha virt. eigenvalues --    1.07880   1.11510   1.17987   1.22828   1.22829
 Alpha virt. eigenvalues --    1.30936   1.52544   1.62083   1.62084   1.69115
 Alpha virt. eigenvalues --    1.96044   2.01846   2.01847   2.10879   2.10880
 Alpha virt. eigenvalues --    2.20515   2.20516   2.31652   2.45984   2.45985
 Alpha virt. eigenvalues --    2.54563   2.61717   2.61717   2.69031   2.77372
 Alpha virt. eigenvalues --    2.77373   2.99734   3.21087   3.86496   3.86497
 Alpha virt. eigenvalues --    4.35392   4.35393   4.36690   4.36690   4.54952
 Alpha virt. eigenvalues --    6.76069   7.76144   7.76146   7.83738   9.85669
 Alpha virt. eigenvalues --   24.03809  26.02183  26.02183  26.27560  48.06571
 Alpha virt. eigenvalues --  215.83434 289.96748 289.96748 290.043151020.83280
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.118193   0.421160   0.421165   0.421145   0.017705  -0.049872
     2  H    0.421160   0.512328  -0.032992  -0.033004  -0.026685  -0.025330
     3  H    0.421165  -0.032992   0.512326  -0.033001  -0.026692  -0.025332
     4  H    0.421145  -0.033004  -0.033001   0.512335  -0.026665  -0.025324
     5  Br   0.017705  -0.026685  -0.026692  -0.026665  35.749672   0.017428
     6  Cl  -0.049872  -0.025330  -0.025332  -0.025324   0.017428  17.607735
 Mulliken charges:
               1
     1  C   -0.349496
     2  H    0.184523
     3  H    0.184526
     4  H    0.184514
     5  Br  -0.704762
     6  Cl  -0.499305
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.204067
     5  Br  -0.704762
     6  Cl  -0.499305
 Electronic spatial extent (au):  <R**2>=           1140.9439
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0005    Y=             -0.0001    Z=             -1.9613  Tot=              1.9613
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.9223   YY=            -43.9222   ZZ=            -84.9143
   XY=             -0.0000   XZ=              0.0007   YZ=             -0.0002
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             13.6640   YY=             13.6641   ZZ=            -27.3281
   XY=             -0.0000   XZ=              0.0007   YZ=             -0.0002
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.0987  YYY=              0.1116  ZZZ=           -121.8755  XYY=             -1.0981
  XXY=             -0.1118  XXZ=            -14.3754  XZZ=              0.0007  YZZ=             -0.0002
  YYZ=            -14.3753  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -71.8778 YYYY=            -71.8783 ZZZZ=          -1702.9962 XXXY=             -0.0001
 XXXZ=              0.7750 YYYX=              0.0002 YYYZ=              0.0780 ZZZX=              0.0043
 ZZZY=             -0.0013 XXYY=            -23.9599 XXZZ=           -250.2545 YYZZ=           -250.2545
 XXYZ=             -0.0788 YYXZ=             -0.7720 ZZXY=             -0.0000
 N-N= 1.713831030512D+02 E-N=-7.695671221774D+03  KE= 3.069966291897D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE729\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.956842,-0.000059,0.000197\H,0,-0.894849,-0.909125,-0.566353\H,0
 ,-0.894833,0.945133,-0.50377\H,0,-0.894893,-0.036195,1.070754\Br,0,1.6
 24244,0.000006,-0.000018\Cl,0,-3.259899,0.000011,-0.000046\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.3880554\RMSD=4.145e-09\Dipole=0.7716
 244,-0.0000534,0.0001864\Quadrupole=-20.3177504,10.1589312,10.1588192,
 0.0001252,-0.0004894,-0.0000064\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 IT PROBABLY IS NOT POSSIBLE FOR A SCIENTIST TO INTERROGATE NATURE 
 FROM A WHOLLY DISINTERESTED VIEWPOINT.
 EVEN IF HE HAS NO PARTICULAR AXE TO GRIND, HE IS LIKELY TO HAVE 
 A DISTINCTIVE WAY OF VIEWING NATURE.

                                                   -- JOHN LOSEE
 Job cpu time:       0 days  0 hours  0 minutes 16.9 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  9.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:47:34 2020.
